packages/plugin-ebarimt-ui/src/components/PerSettings.tsx

```
@@ -245,10 +253,50 @@ const PerSettings: React.FC<Props> = (props: Props) => {
           </FlexBetween>
           {renderInput('defaultGSCode', 'default united code', '')}
           {renderInput('branchNo', 'branch No', '')}
-          {renderCheckbox('hasVat', 'has Vat', '')}
-          {renderInput('vatPercent', 'vat Percent', '')}
-          {renderCheckbox('hasCitytax', 'has Citytax', '')}
-          {renderInput('cityTaxPercent', 'cityTax Percent', '')}
+          <FlexRow>
+            {renderCheckbox('hasVat', 'has Vat', '')}
+            {state.config.hasVat && renderInput('vatPercent', 'vat Percent', '') || <></>}
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.

(lint/complexity/noUselessFragments)